ReactiveCocoa/ControlSubject.swift

```diff
@@ -0,0 +1,44 @@
+import ReactiveSwift
+import enum Result.NoError
+
+/// Represents a subject of an interactive control.
+///
+/// Generally speaking, `ControlSubject` emits changes only for the user
+/// interactions in which the subject specializes.
+public final class ControlSubject<Value> {
```

Comment:
That all makes sense.

But the pragmatism of conforming to `PropertyProtocol` might be preferable to the ideal of not asking controls for a value.

Or maybe this just needs a better name to really clarify the concept.